### DISCUSSION TITLE
Problems and suggestions with swahili translation

### DISCUSSION THREAD

**Discussion by Wycklife** [IDEMSInternational/R-Instat#6614] [Ideas]
We will be posting suggestions and the errors we find on the swahili translation here.

1. Going over the already existing translation and confirming if the language used is correct.
2. Checking if the translated words have any effect on the dialogs.

**Comment by berylwaswa** [comment]
Here are a few suggestions and corrections for the language used on the Menus
File
- Save As - **Hifadhi ila Kama** not Okoa kama
- More - **Zaidi**
- Exit - **Toka** not Utgang

Edit
- Paste - **Bandika** not Baandika
- Paste New Data Frame - **Bandika Sura Mpya ya Takwimu**

Prepare
- Column: Factor - **Safuwima: Nambari Kamilifu** not Safuwima: Sababu
- Data Objects - **Vitu vya Takwimu** not Kitu cha Takwimu

Describe
- One Variable - **Kiegezo Kimoja**
- Specific - **Maalum**
- General - **Jumla** not Mkuu
- Multivariate - **Anuwai badilika**

Model 
- Model - **Mfano**

Structured 
- Low-flow - **Mtiririko Mdogo**
- Survival - **Kuishi** not Kuokoka

I started by going through all the menus in there. Here are more words;

File Menu
**Save As**

- Save Data As - Hifadhi Data Kama
- Save Output As - Hifadhi Matokeo Kama
- Save Log As - Hifadhi Kumbukumbu Kama

Export

Export R Workspace - Hamisha Nafasikazi ya R

Prepare Menu
**Data Frame**
- Row Numbers/ Names - **Nambari za/ Majina ya Safu Mlalo**
- Color by Property - **Rangi kwa Sifa**

**Check Data**
- Duplicates - **Marudio**
- Box plots
- One Variable Summarise - **Muhtasari wa Kiegezo Kimoja**
- One Variable Graph - **Grafu ya Kiegezo Kimoja**
- One Variable Frequencies - **Masafa ya Kiegezo Kimoja**
- Export to OpenRefine - **Hamisha kuenda OpenRefine**
- Jitter - **Mtetemo**
- Prepare to Share(sdc package) - **Jitayarishe kugawa (kifurushi cha sdc)**
- Anonymize ID Column - **Ficha jina la Safu**

Column: Calculate
- Regular Sequence - **Mpangilio wa Kawaida**
- Duplicate Column - **Nakili Safu**
- Polynomials - **Uzidishaji Anuwai**
- Permute Columns - **Panga Safu**

**Comment by rdstern** [comment]
@berylwaswa that's great to have a copy like this.  But you now need to accept that these suggestions now need to be acted on.  So, either you should nominate someone in the message who can discuss with you, or you should (provisionally) make those corrections yourselves.  Then they will result in a pull request, and once accepted, they mill be implemented in the next version.

**Comment by berylwaswa** [comment]
@rdstern we will discuss the way forward and we may work on those
corrections ourselves.
Kind Regards,
Beryl Waswa,
Research Assistant,
INNODEMS.

**Comment by berylwaswa** [comment]
@Wycklife and @berylwaswa will work on making these corrections once the French translations are done, in the meantime, I will keep adding the list of suggestions for the translation.

**Comment by shadrackkibet** [comment]
Good that progress is being made on Kiswahili translation. I wonder how efficient it is to wait for the completion of the French translation before doing anything on the Kiswahili translation. Can there be a closer collaboration between groups that work on translation such that this is done at the same time? For example, resizing of controls can be done at the same time while switching between languages. Another option is to work on the same PR which requires coordination between groups/individuals. 

This will means more work but ideally, it will consolidate translation work and therefore avoid multiple PRs and delays.

My thoughts.

**Comment by berylwaswa** [comment]
Roger suggested we wait but if you think there is a way we can work simultaneously then we can do that. @Wycklife let me know what and how we can do that.
